app/app.js

```diff
@@ -1,60 +1,70 @@
-var app = angular.module("myCalculator", []);
+var app = angular.module("myCalculator", ['ngRoute']);
+app.config(['$routeProvider', function($routeProvider){
+	$routeProvider.when('/', {
+		templateUrl : 'templates/home.html',
+		controller : 'CalculatorCtrl'
+	}).when('/newMeal', {
+		templateUrl : 'templates/new-meal.html',
+		controller: 'CalculatorCtrl'
+	}).when('/earnings', {
+		templateUrl: 'templates/my-earnings.html',
+		controller: 'CalculatorCtrl'
+	});
+}]);
+app.controller('CalculatorCtrl', ['$scope', function($scope){
 
-app.controller("CalculatorCtrl", function(){
-
-	this.initCharge = function (){
-		this.subtotalCharge = 0;
-		this.tipCharge = 0;
-		this.totalCharge = 0;	
+	$scope.initCharge = function (){
+		$scope.subtotalCharge = 0;
+		$scope.tipCharge = 0;
+		$scope.totalCharge = 0;	
 	};
 
-	this.initEarnings = function(){
-		this.tipTotal = 0;
-		this.mealCount = 0;
-		this.tipAverage = 0;
+	$scope.initEarnings = function(){
+		$scope.tipTotal = 0;
+		$scope.mealCount = 0;
+		$scope.tipAverage = 0;
 	};
 
 	//Trigered by Cancel button
-	this.cancelValues = function(){
-	this.price = "";
-	this.tax = "";
-	this.tip ="";
+	$scope.cancelValues = function(){
+	$scope.price = "";
+	$scope.tax = "";
+	$scope.tip ="";
```

Comment:
Indenting the inside of this function would make it easier to read. It took me a second to realize these were inside cancelValues..